.doxygen

```diff
@@ -843,7 +843,6 @@ FILE_PATTERNS          = *.c \
                          *.inc \
                          *.m \
                          *.markdown \
-                         *.md \
```

Comment:
@julianoes Currently our Makefile runs doxygen over the /install directory. First time this runs, everything works fine. Second time it runs the directory contains markdown files (that I created). These get parsed by doxygen creating xml ... and so on.
This is easiest solution to problem. I don't know if it has any "costs" - if not the alternative is to run doxygen only on /install/includes ... which I'd leave to you to do :-)